config/locales/de.yml

```diff
@@ -0,0 +1,182 @@
+de:
+  activerecord:
+    errors:
+      models:
+        user:
+          attributes:
+            email:
+              blank: darf nicht leer sein
+              taken: ist schon vergeben
+              invalid: ist ungültig
+            role:
+              inclusion: ist nicht in der Liste enthalten
+            password:
+              blank: darf nicht leer sein
+              too_short: zu kurz (min. 6 Zeichen)
+            password_confirmation:
+              confirmation: Die eingegebenen Passwörter stimmen nicht überein
+        profile:
+          attributes:
+            first_name:
+              blank: darf nicht leer sein
+            last_name:
+              blank: darf nicht leer sein
+            avatar_content_type:
+              invalid: ist ungültig
+    models:
+      profile: Profil
+      user: Benutzer
+    attributes:
+      user:
+        email: Email
+        role: Rolle
+      profile:
+        first_name: Vorname
+        last_name: Nachname
+        phone: Telefonnummer
+        avatar: Bild
+        address: Adresse
+        profession: Beruf
+        monday: Montag
+        tuesday: Dienstag
+        wednesday: Mittwoch
+        thursday: Donnerstag
+        friday: Freitag
+  devise:
+    confirmations:
+      confirmed: Vielen Dank für Ihre Registrierung. Bitte melden Sie sich jetzt an.
+      send_instructions: Sie erhalten in wenigen Minuten ein Email, mit dem Sie Ihre Registrierung bestätigen können.
+      send_paranoid_instructions: Falls Ihre Mailadresse in unserer Datenbank existiert erhalten Sie in wenigen Minuten eine Email mit der Sie Ihre Registrierung bestätigen können.
```

Comment:
Falls Ihre Mailadresse in unserer Datenbank existiert, erhalten Sie in wenigen Minuten eine Email, mit der Sie Ihre Registrierung bestätigen können.

> two commas missing